esx_service/cli/vmdkops_admin.py

```diff
@@ -123,7 +127,7 @@ def commands():
                     'metavar': 'Col1,Col2,...'
                 },
                 '--tenant' : {
-                    'help': 'Displays VMs for a given tenant'
+                    'help': 'Displays volumes for a given tenant'
```

Comment:
unfortunately -c option does not filter rows, it simple shows less columns, so there is currently no way to scope the list to a specific tenant (other that `| grep`)